### ISSUE TITLE
p2:site is wrong shortcut for goal

### ISSUE THREAD

**Issue by Bananeweizen** [reficio/p2-maven-plugin#172]
The readme (and maybe other places) mentions to run p2:site. For basically all users that doesn't work. It would require to have a maven prefix defined to map the full plugin name to the prefix "p2". The readme should rather say to invoke "org.reficio:p2-maven-plugin:site" instead. That will work for everyone, without defining prefixes first.

**Comment by bmarwell** [comment]
I would agree, but now I am curious why the integration tests work then. I have not found a mapping.
